person.rb

```diff
@@ -2,15 +2,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :rental
+  attr_accessor :name, :age, :rentals
 
   def initialize(id = rand(1...100), name = 'Unknown', age = nil, parent_permission: true)
     super()
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
-    @rental = []
+    @rentals = []
```

Comment:
- [ ] In the `Teacher` class I suggest that in the initializing method change it to 
```
 def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end
  
  ```